src/components/q&a/QuestionsList/QuestionsList.jsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import IndividualQuestion from './IndividualQuestion/IndividualQuestion';
+import BottomBar from './BottomBar/BottomBar';
+import styles from './QuestionsList.css';
+
+class QuestionsList extends React.Component {
```

Comment:
Maybe try to refactor this to a functional component, using the `useState` hook?